dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from sklearn.metrics import mean_squared_error
</pre><p>please combine the code blocks above into a single click for model definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>